syncserver-settings/src/lib.rs

```diff
@@ -38,6 +36,12 @@ pub struct Settings {
     pub cors_allowed_methods: Option<Vec<String>>,
     pub cors_allowed_headers: Option<Vec<String>>,
 
+    /// The maximum number of blocking threads that can be used by the worker.
+    /// Note, we don't want "None" here because we use this as part of the
+    /// metric periodic reporter. The default value is 512. Setting the value to
+    /// `0` will
+    pub worker_max_blocking_threads: usize,
```

Comment:
I think this comment got cut off?

So AFAICT we could still use `Option` here, we would just need to set the default to `None`, but I don't really mind the non Option w/ `0` instead.

Reply:
Yeah, I'll remove the `0` comment, since it gets really convoluted what happens if we specify 0. 
In short, we should never specify 0.